.github/scripts/tkn_check_parse.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# removes `actions's` default -e flag as it breaks the logic
+set +e
+
+setup() {
+    curl -LO "https://github.com/tektoncd/cli/releases/download/v${TKN_CLI_VERSION}/tektoncd-cli-${TKN_CLI_VERSION}_Linux-64bit.deb" &>/dev/null
+    sudo dpkg -i ./tektoncd-cli-${TKN_CLI_VERSION}_Linux-64bit.deb >/dev/null
+}
+
+is_yaml() {
+    FILE=`basename $1`
+    DIRNAME=`dirname $1`
+    [ ${DIRNAME##*/} == "workflows" ] && return 1
```

Comment:
What's this for? The only `workflows` dir we have is `.github/workflows`, right? And this will not be included in the changeset because of the pattern `catalog/*/*/*/*.yaml` in the github action.

Reply:
Indeed. I added a test file and afterwards removed it in favor of a small test change in an actual yaml. I will remove this part as well.

Reply:
OK, cool.